engine/installation/linux/docker-ce/centos.md

```diff
@@ -77,31 +77,38 @@ from the repository.
     ```
 
 2.  Use the following command to set up the **stable** repository. You always
-    need the **stable** repository, even if you want to install **edge** builds
-    as well.
+    need the **stable** repository, even if you want to install builds from the
+    **edge** or **testing** repositories as well.
```

Comment:
I think with the upcoming changes, there's no longer a need to have them both (@andrewhsu?) as stable releases will be pushed to all of _edge_, _stable_, and _testing_